test/features/bootstrap_1.feature

```diff
@@ -0,0 +1,28 @@
+@bootstrap
+Feature: HA bootstrap process 1
```

Comment:
We need to add better names to the feature. I guess each feature has a purpose, so this should be described better if possible